RobotBase.java

```diff
@@ -54,6 +68,21 @@ public RobotContainerBase getRobotContainer() {
    */
   @Override
   public void robotInit() {
+    Logger.getInstance().recordMetadata("ProjectName", "2023-Robot"); // Set a metadata value
+
+    if (isReal()) {
+        Logger.getInstance().addDataReceiver(new WPILOGWriter("/home/lvuser")); // Log to a USB stick
+        Logger.getInstance().addDataReceiver(new NT4Publisher()); // Publish data to NetworkTables
```

Comment:
we should keep an eye on how much overhead this adds. we may not want it in matches